TreeMaker/python/doLostLeptonBkg.py

```diff
@@ -8,18 +8,23 @@ def doLostLeptonBkg(self,process,METTag):
             pfCandsTag  = cms.InputTag('packedPFCandidates')
         )
 
+    from TreeMaker.TreeMaker.TMEras import TMeras
     from TreeMaker.Utils.isolationproducer_cfi import isolationproducer
     process.IDMuonMiniIso = isolationproducer.clone(
         LeptonTag = cms.InputTag('LeptonsNew:IdMuon'),
-        LeptonType = cms.string('muon'),
-        PFCandTag = cms.InputTag('packedPFCandidates'),
-        JetTag = cms.InputTag('HTJets')
+        LeptonType = cms.string('muon')
+    )
+    TMeras.TM2017.toModify(process.IDMuonMiniIso,
+        electronEAValues = cms.vdouble(0.1566, 0.1626, 0.1073, 0.0854, 0.1051, 0.1204, 0.1524),

```

Comment:
these values are duplicated several times in this file - define them once and then assign multiple times?